Automattic/wp-calypso
# Fix owned Complete plan displaying twice on the pricing page.

This PR fixes a bug where the Jetpack Complete bundle card was displaying twice on the pricing page for logged-in users when their site has a currently active subscription to Jetpack Complete. 

Github Task: https://github.com/orgs/Automattic/projects/724/views/1?pane=issue&itemId=37486302

**BEFORE**

![Image](https://github.com/Automattic/jetpack-marketing/assets/11078128/090cfefe-d6c0-4880-b85e-ac9b158ffb8c)




## Proposed Changes

* Fixes the way we get all the plan slugs for each of the billing terms of the current plan.  We were only getting yearly and monthly plan slugs, but now with the addition of bi-yearly plans, there was a bug where only monthly and bi-yearly were being returned.
* Also a bug was fixed where the Complete Bi-yearly plan term was set to TERM_ANNUALLY instead of TERM_BIENNIALLY.

## Testing Instructions



- Checkout this PR and spin up the Calypso Green env, -OR:
- Or click on the Jetpack Cloud Live (direct link) below.
- With a connected JN or Ephemeral site, go to `/pricing/:SITE` (logged-in) and purchase Jetpack Complete.
- After purchase completed, go back to `/pricing/:SITE` (logged-in) and verify that the Jetpack Complete bundle card displays as "Active on your site" and it only displays once. (See screenshot below).
- You can pull up https://cloud.jetpack.com/pricing/:SITE (logged-in) to view the regression. (The Complete card displaying twice (See screenshot above)).

**AFTER**

![Markup 2023-09-08 at 17 13 09](https://github.com/Automattic/wp-calypso/assets/11078128/d63de8db-9469-44f8-8ab9-b9085b7ae7b9)


## Pre-merge Checklist



- [X ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request chanThis PR fixes a bug where ges what data or activity we track or use (p4TIVU-ajp-p2)?
